### PR TITLE
Pot/Potluck/Potman updates

### PR DESCRIPTION
**Pull request by hny-gd**

### CODE REVIEW

File: 2021q3/pot.adoc [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+link:https://github.com/hny-gd/potluck[Potluck on github] URL: link:https://github.com/hny-gd/potluck[https://github.com/hny-gd/potluck] +
+link:https://github.com/grembo/potman[Potman on github] URL: link:https://github.com/grembo/potman[https://github.com/grembo/potman]
+
+Contact: Luca Pizzamiglio (Pot) <pizzamig@freebsd.org> +
+Contact: Stephan Lichtenauer (Potluck) <sl@honeyguide.eu> +
+Contact: Michael Gmelin (Potman) <grembo@FreeBSD.org>
+
+Pot is a jail management tool that link:https://www.freebsd.org/news/status/report-2020-01-2020-03/#pot-and-the-nomad-pot-driver[also supports orchestration through Nomad].
+
+In the last quarter, a new release link:https://github.com/pizzamig/pot/releases/tag/0.13.0[0.13.0] with a number of major new features has been released. +
```
Comment (pauamma) [inline 2021q3/pot.adoc:15]: "a new release [...] has been released." Perhaps "was made available"? (Minor nit)

File: 2021q3/pot.adoc [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+link:https://github.com/pizzamig/pot/issues/148[Layered images] allow a split into a foundation image component that only changes seldomly and that e.g. contains the basic jail and packages and a "leaf" image component on top with potentially small additions that might change more frequently, like software built in a CI pipeline. Since it is not the complete image that has to be rebuilt each time, image creation and distribution can be sped up immensely. +
```
Comment (pauamma) [inline 2021q3/pot.adoc:16]: s/seldomly/seldom/ (it's already an adverb)

File: 2021q3/pot.adoc [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+Also a link:https://github.com/pizzamig/pot/issues/162[copy-out command] has been included where the container state that was initialised from within the container can be persisted and reinjected into additional containers afterwards.
```
Comment (pauamma) [inline 2021q3/pot.adoc:17]: s/be persisted/be made persistent/ maybe? (Your call. I find this usage understandable but mildly jarring, but I may well be showing my age here.)

File: 2021q3/pot.adoc [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+In the last quarter, a new release link:https://github.com/pizzamig/pot/releases/tag/0.13.0[0.13.0] with a number of major new features has been released. +
+link:https://github.com/pizzamig/pot/issues/148[Layered images] allow a split into a foundation image component that only changes seldomly and that e.g. contains the basic jail and packages and a "leaf" image component on top with potentially small additions that might change more frequently, like software built in a CI pipeline. Since it is not the complete image that has to be rebuilt each time, image creation and distribution can be sped up immensely. +
+Also a link:https://github.com/pizzamig/pot/issues/162[copy-out command] has been included where the container state that was initialised from within the container can be persisted and reinjected into additional containers afterwards.   
+
+Potluck aims to be to FreeBSD and Pot what Dockerhub is to Linux and Docker: A repository of Pot flavours and complete container images for usage with Pot and in many cases Nomad.
```
Comment (pauamma) [inline 2021q3/pot.adoc:19]: s/A repository/a repository/ (since it's not a complete sentence).

File: 2021q3/pot.adoc [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+Here we had a busy quarter. Not only did we commit a number of new images like link:https://potluck.honeyguide.net/blog/nextcloud-nginx-nomad/[Nextcloud] which can be deployed via Nomad, also the images that can be used to build the foundation of a virtual DC (link:https://potluck.honeyguide.net/blog/nomad-server/[Nomad],  link:https://potluck.honeyguide.net/blog/consul/[Consul] and link:https://potluck.honeyguide.net/blog/vault/[Vault]) themselves have received major updates.
```
Comment (pauamma) [inline 2021q3/pot.adoc:21]: s/also the images that can be used/but the images used/
"DC"? data center?
Spurious space before "link:https://potluck.honeyguide.net/blog/consul/[Consul]"

File: 2021q3/pot.adoc [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+For these images, further improvements for even better security and reliability will likely be finished in the coming quarter. 
+
+Also, we are aware that right now the advantages of the container approach as it is described in link:https://honeyguide.eu/posts/virtual-dc1/[Virtual DC Part I]/link:https://honeyguide.eu/posts/virtual-dc2/[Part II]/link:https://honeyguide.eu/posts/virtual-dc3/[Part III] is not yet obvious and transparent enough and also no longer completely up to date, so we plan to provide additional documentation as soon as we find the time to do so.
```
Comment (pauamma) [inline 2021q3/pot.adoc:25]: "advantages [...] are not yet obvious" or "advantage [...] is not yet obvious" (number agreement)

File: 2021q3/pot.adoc [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+
+For these images, further improvements for even better security and reliability will likely be finished in the coming quarter. 
+
+Also, we are aware that right now the advantages of the container approach as it is described in link:https://honeyguide.eu/posts/virtual-dc1/[Virtual DC Part I]/link:https://honeyguide.eu/posts/virtual-dc2/[Part II]/link:https://honeyguide.eu/posts/virtual-dc3/[Part III] is not yet obvious and transparent enough and also no longer completely up to date, so we plan to provide additional documentation as soon as we find the time to do so. 
+
+Potman aims to simplify building Pot images with Vagrant and VirtualBox based on the Potluck approach, e.g. as part of a DevOps workflow for software development including testing and publishing them to a repository.
+
+Here, not too much has happened over the last quarter but the current idea is to in the medium to longer term incorporate additional features to modularise and simplify the image build definitions and then utilise Potman in the Potluck library build process.
```
Comment (pauamma) [inline 2021q3/pot.adoc:29]: I'd move "in the medium to longer term" to after "additional features".

**Comment by hny-gd** [comment]
Thank you very much @pauamma, your feedback was very much appreciated, especially since English is not my first language. :-)
I have merged all of your comments (and I hope I have not introduced new mistakes...)

**Comment by pauamma** [comment]
You're welcome. Looks good to me now.

**Review by pauamma — APPROVED** [review]

**Pull request merged**